src/tools/cephfs/cephfs-shell

```diff
@@ -1556,10 +1569,10 @@ if __name__ == '__main__':
     sys.argv.append(exe)
     sys.argv.extend([i.strip() for i in ' '.join(args.commands).split(',')])
 
-    setup_cephfs(config_file)
+    setup_cephfs(ceph_conf)
```

Comment:
so `ceph_conf` is alway an empty string ?

Reply:
Incorporated what you suggested on IRC (got rid of the variable).